dashboard/app/views/levels/_free_response.html.haml

```diff
@@ -58,3 +55,6 @@
   -# Don't render the dialog partial if we're inside a LevelGroup.
   = render partial: 'levels/dialog', locals: {app: 'free_response'} unless in_level_group
   = render partial: 'levels/teacher_markdown', locals: {data: {'teacher_markdown' => level.solution}} unless is_contained_level
+
+- content_for(:head) do
+  %script{src: webpack_asset_path('js/levels/_free_response.js'), data: {freeresponse: js_data.to_json}}
```

Comment:
why does the script need to be loaded in a `content_for` block here?

Reply:
It doesn't have to.  I have a personal preference script tags in `<head>` (or a distaste for dropping them at the end of the body in lieu of a proper ready handler) but if I'm being honest I don't have a technical justification for that and this script will work correctly from any position in the DOM.  Would you prefer that I drop the `content_for` block here?

Reply:
ohh, that makes sense. it looks like we commonly use both patterns throughout the codebase, so i don't think i have a strong preference on this one